addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1933,6 +1933,16 @@ var SnippetsMenu = Widget.extend({
         // Force non editable part to contentEditable=false
         $html.find('.o_not_editable').attr('contentEditable', false);
 
+        // TODO remove me in 16.0: introduced in a 14.0 fix to allow users to
+        // switch between the different tabs of a custom tabs even in the
+        // editor mode. Before this fix, it was not possible because the
+        // elements of the tabs located on the website and the elements of the
+        // save tabs located on the editor had the same id's so the anchors
+        // were referring to the wrong elements.
+        for (const $customTabPane of $html.find('#snippet_custom_body .tab-pane')) {
+            $customTabPane.id = "";
```

Comment:
- `$customTabPane` here is not a jQuery element but a standard HTMLElement. For those variables, we don't name them starting by `$` and I strongly suggest naming them by suffixing them by `El`. So here: `customTabPaneEl`.
- In fact, why not doing `.removeAttribute('id')` instead of setting it to an empty string? It does not really matter here but `id=""` is actually invalid HTML.